source/Orchestrator/send_notifications.py

```diff
@@ -37,6 +37,15 @@ def format_details_for_output(details):
 
     return details_formatted
 
+def set_message_prefix_and_suffix(event):
```

Comment:
Moved this into its own function, manually verified that it changes the message as expected. I wanted to write a unit test for this specifically, but it was difficult without making logical changes to the code.